AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/routes/v1/ReportsRoutesBase.java

```diff
@@ -100,9 +100,11 @@ public Object deleteReport(Request req, Response res, int id) throws Exception {
     public List<T> searchReports(Request req, Response res) {
         permissionsVerifier.verifyAdminPermission(req);
 
-        return reportsDao.searchReports(parseSearchCriteria(req))
+        List<T> test = reportsDao.searchReports(parseSearchCriteria(req))
```

Comment:
Could you put this back to how it was? Don't want to leave a random variable called `test` floating around